.kokoro/dependencies.sh

```diff
@@ -47,11 +47,11 @@ function completenessCheck() {
   # This is stripped from the output as it is not present in the flattened pom.
   # Only dependencies with 'compile' or 'runtime' scope are included from original dependency list.
   msg "Generating dependency list using original pom..."
-  mvn dependency:list -f pom.xml -DincludeScope=runtime -Dsort=true | grep '\[INFO]    .*:.*:.*:.*:.*' | sed -e 's/ --.*//' >.org-list.txt
+  mvn dependency:list -f pom.xml -DincludeScope=runtime -DexcludeArtifactIds=gson,commons-codec,commons-logging,opencensus-contrib-http-util,httpclient,httpcore -Dsort=true | grep '\[INFO]    .*:.*:.*:.*:.*' | sed -e 's/ --.*//' >.org-list.txt
```

Comment:
Where is this exclusion list coming from? A template?

Reply:
Yes indeed. It comes from synthtool. There is a bug in maven-dependency-plugin which is causing this issue: https://issues.apache.org/jira/browse/MDEP-737 
So we temporarily add this exclusion as a workaround.